modules/apps/oauth2-provider/oauth2-provider-service/src/main/java/com/liferay/oauth2/provider/service/impl/OAuth2ScopeGrantLocalServiceImpl.java

```diff
@@ -183,4 +214,7 @@ private boolean _isMatch(
 		return true;
 	}
 
+	@Reference
+	private OAuth2AuthorizationLocalService _oAuth2AuthorizationLocalService;
```

Comment:
This is not used anywhere.

Reply:
Resolved now.  Strange SF didn't pick that up though.